frontend/package.json

```diff
@@ -125,7 +125,7 @@
     "@patternfly/quickstarts": "1.1.0",
     "@patternfly/react-catalog-view-extension": "4.12.38",
     "@patternfly/react-charts": "6.15.14",
-    "@patternfly/react-core": "4.147.2",
+    "@patternfly/react-core": "4.152.4",
```

Comment:
required update, as the `TreeView` component did not provide the `hasGuides` property in the previous version (even if it's part of the Patternfly specification)